test/testdata/rewriter/t_struct/inexact.rb.rewrite-tree.exp

```diff
@@ -50,4 +50,28 @@ class <emptyTree><<C <root>>> < (::<todo sym>)
   <emptyTree>::<C Inexact>.new({:"foo" => 3, :"bar" => "hey"})
 
   <emptyTree>::<C Inexact>.new({:"foo" => false, :"quz" => [1, 2, 5]})
+
+  class <emptyTree>::<C Child><<C <todo sym>>> < (<emptyTree>::<C Inexact>)
+    ::T::Sig::WithoutRuntime.sig() do ||
+      <self>.params({}).returns(<emptyTree>::<C Symbol>)
+    end
+
+    def qux<<C <todo sym>>>(&<blk>)
+      ::T.unsafe(::Kernel).raise("Sorbet rewriter pass partially unimplemented")
+    end
+
+    ::T::Sig::WithoutRuntime.sig() do ||
+      <self>.params({:"arg0" => <emptyTree>::<C Symbol>}).returns(<emptyTree>::<C Symbol>)
+    end
+
+    def qux=<<C <todo sym>>>(arg0, &<blk>)
+      ::T.unsafe(::Kernel).raise("Sorbet rewriter pass partially unimplemented")
+    end
```

Comment:
When we don't see the super class, again we make neither the getter nor the setter.